character/controllers/ping_sound_controller.gd

```diff
@@ -0,0 +1,57 @@
+extends ControllerBase
+class_name PingSoundController
+
+## Minimum impulse to cause a ping sound
+## Note that this is based on current gravity and mass,
+##  but was determined through testing
+@export
+var accel_ping_threshold:float = 25
+## Impulse which corresponds to the max ping volume
+@export
+var accel_ping_max:float = 200
+
+## Minimum ping volume gain, in Db
+@export
+var min_ping_gain:float = -10
+## Maximum ping volume gain, in Db
+@export
+var max_ping_gain:float = 5
+
+## Minimum ping pitch scale
+@export
+var min_ping_pitch:float = 0.8
+## Maximum ping pitch scale
+@export
+var max_ping_pitch:float = 1.2
+
+var last_velocity:Vector2
+
+func _on_ready():
+	last_velocity = linear_velocity
+
+func _physics_process(delta) -> void:
+	var accel_magnitude := (linear_velocity - last_velocity).length()
+	last_velocity = linear_velocity
+	
+	if accel_magnitude > accel_ping_threshold:
```

Comment:
Basing this only on accel means that there is a ping sound when using new pull-back movement. This is fine for now, but once we add a sound effect specifically for the pull-back (not until we want to juice things up later), we should update this to ensure it's not from the boost.